shared/api/network_addresses.go

```diff
@@ -7,14 +7,14 @@ package api
 //
 // API extension: network_allocations.
 type NetworkAllocations struct {
-	// The CIDR subnet of the forward
+	// The network address of the allocation (in CIDR format)
 	// Example: 192.0.2.1/24
-	Addresses []string `json:"addresses" yaml:"addresses"`
+	Address string `json:"addresses" yaml:"addresses"`
```

Comment:
That's an API break, isn't it?

Reply:
It would be if it had been released. But it hasn't yet.

Reply:
It was added in this unreleased PR https://github.com/canonical/lxd/pull/11831/commits/7b53f9bfc5a2a5a4f68406096137eb3a3c1b8e10